File: Problem Set 1 - C/Cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float cash;
    int change;
    int count = 0; //This variable allow us to count the number of change that we should return.
    do
    {
        cash = get_float("Cash owed: "); // Getting the amount of money to change.
    }
    while (cash <= 0);
    change = round(cash * 100); //Rounding the cents to the nearest penny(integer).

    while (change >= 25) //This while loop allows us to use as many 25 coins as possible.
    {
        change = change - 25;
        count++;
    }
    while (change >= 10)//This while loop allows us to use as many 10 coins as possible.
    {
        change = change - 10;
        count++;
    }
    while (change >= 5)//This while loop allows us to use as many 5 coins as possible.
    {
        change = change - 5;
        count++;
    }
    while (change >= 1)//This while loop allows us to use as many 1 coins as possible.
    {
        change = change - 1;
        count++;
    }
    printf("%i\n", count);
}

File: Problem Set 6 - Python/Cash/cash.py
from cs50 import get_float # importing the get_float function from the cs50 library.

change = get_float("Change owed: ") # Ask the user for the input.
count = 0
while change < 0:
    change = get_float("Change owed: ")
    change = round(change * 100)

change = round(change * 100)

while change >= 25:
    change -= 25
    count += 1

while change >= 10:
    change -= 10
    count += 1

while change >= 5:
    change -= 5
    count += 1

while change >= 1:
    change -= 1
    count += 1

print(count) # print the final result.
